container/exceptions.py

```diff
@@ -5,9 +5,11 @@
 
 logger = logging.getLogger(__name__)
 
-class AnsibleContainerNotInitializedException(Exception):
+class AnsibleContainerException(Exception):
```

Comment:
If you're going to do this, have the other exceptions derive from `AnsibleContainerException`